recipes/libbezier/bld.bat

```diff
@@ -0,0 +1,26 @@
+:: NOTE: This assumes the following environment variables have been set.
+::       - `%SUBDIR%`
+::       - `%c_compiler%`
+::       - `%LIBRARY_PREFIX%`
+:: H/T: https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html
+
+mkdir build
```

Comment:
I took my inspiration from the `libpng`-provided [`bld.bat`][1], which used

```
mkdir build-%SUBDIR%-%c_compiler%
```

I couldn't find any mention of `SUBDIR` or `c_compiler` (or `f_compiler`) in the [environment variables][2] doc, so am not sure what would be the best move here. (Similar question applies to the build directory created in `build.sh`.)

[1]: https://github.com/conda-forge/libpng-feedstock/blob/13bf3cf2b80c996c1bb0f32695b246362b047d40/recipe/bld.bat
[2]: https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html